src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node head;
+    private Node tail;
+    private int size;
+
+    private class Node {
```

Comment:
[Private methods and classes should be after public ones in your class.](https://github.com/Kalixone/jv-linked-list/blob/develop/checklist.md#private-methods-and-classes-should-be-after-public-ones-in-your-class)